### DISCUSSION TITLE
opam error

### DISCUSSION THREAD

**Discussion by gAlleb** [savonet/liquidsoap#3400] [Installation] [answered]
Hi, since liq 2.2.1 appeared recently on opam, i've tried to upgrade from liq 2.2.0
This is what I get:
```
$ opam update
$ opam upgrade liquidsoap
[ERROR] Package conflict!
  * No agreement on the version of liquidsoap-libs-extra:
    - liquidsoap >= 2.2.1 → liquidsoap-libs-extra != 2.2.0
 ```
What should i do? Any help would be appreciated. Thanks!

Maybe it is somehow related to this conversation https://github.com/ocaml/opam-repository/pull/24356 and what was initially `liquidsoap-libs-extra.2.2.1~1`

**Comment by smimram** [comment]
Does it help if you try updating both at the same time?
```
opam upgrade liquidsoap-libs-extra liquidsoap
```

**Reply by gAlleb** [reply]
Nope, answer from opam is the same.

**Reply by gAlleb** [reply]
Also I've noticed some strange icons here https://opam.ocaml.org/packages/liquidsoap/
<img width="917" alt="Снимок экрана 2023-09-11 в 14 09 23" src="https://github.com/savonet/liquidsoap/assets/144447266/f2cd6c6f-2634-4549-acc5-7ff2fbcb0e18">

**Reply by vitoyucepi** [reply]
- opam - master - liquidsoap - 2.2.0
  https://github.com/ocaml/opam-repository/blob/16b64306691dcfcca776bed47b86bbb63d54b09d/packages/liquidsoap/liquidsoap.2.2.0/opam#L17-L23

- opam - master - liquidsoap - 2.2.1
  https://github.com/ocaml/opam-repository/blob/16b64306691dcfcca776bed47b86bbb63d54b09d/packages/liquidsoap/liquidsoap.2.2.1/opam#L17-L23

- liquidsoap - v2.2.1
  https://github.com/savonet/liquidsoap/blob/ea3b8bca9927ade41e487b83e2b6a824655ba232/liquidsoap.opam#L18-L24

- liquidsoap - rolling-release-v2.2.x
  https://github.com/savonet/liquidsoap/blob/7e014f02ed27eb006a90ec504f19587e48113b81/liquidsoap.opam#L18-L24

**Reply by toots** [reply, marked as answer]
Thanks for reporting. Fix is pending here: https://github.com/ocaml/opam-repository/pull/24403
